scripts/pi-hole/php/header.php

```diff
@@ -176,7 +176,7 @@ function pidofFTL()
 <head>
     <meta charset="utf-8">
     <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://api.github.com; script-src 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'">
-    <title>Pi-hole Admin Console</title>
+    <title>Pi-hole - <?php echo gethostname() ?></title>
```

Comment:
Last nit to pick and I'm not sure if it's possible: Can `gethostname()` return nothing, or fail? If that is a possibility, can the hyphen be conditional on the echo returning something? I'd like to avoid having the output look like the sample below:

`Pi-hole - `